src/content-handlers/iiif/modules/uv-dialogues-module/ShareDialogue.ts

```diff
@@ -161,98 +155,87 @@ export class ShareDialogue<
       '<option value="large" data-width="800" data-height="600">800 x 600</option>'
     );
     this.$customSizeDropDown.append(
-      '<option value="custom">' + this.content.customSize + "</option>"
+      `<option value="custom">${this.content.customSize}</option>`
     );
+    this.$customSizeDropDown.change(() => {
+      this.update();
+    });
+    this.$customSize.append(this.$customSizeDropDown);
 
     this.$widthInput = $(
-      '<input class="width" type="text" maxlength="10" aria-label="' +
-        this.content.width +
-        '"/>'
+      `<input class="width" type="text" maxlength="10" aria-label="${this.content.width}"/>`
     );
+    this.$widthInput.on("keydown", (e) => {
+      return Numbers.numericalInput(e);
+    });
+    this.$widthInput.change(() => {
+      this.updateHeightRatio();
+      this.update();
+    });
+
     this.$customSize.append(this.$widthInput);
+    this.$embedSection.append(this.$customSize);
 
+    // WIDTH x HEIGHT
     this.$x = $('<span class="x">x</span>');
     this.$customSize.append(this.$x);
 
     this.$heightInput = $(
-      '<input class="height" type="text" maxlength="10" aria-label="' +
-        this.content.height +
-        '"/>'
-    );
-    this.$customSize.append(this.$heightInput);
-
-    const iiifUrl: string = this.extension.getIIIFShareUrl(
-      this.shareManifestsEnabled
+      `<input class="height" type="text" maxlength="10" aria-label="${this.content.height}"/>`
     );
-
-    if (this.shareManifestsEnabled) {
-      this.$iiifButton = $(
-        '<a class="imageBtn iiif" href="' +
-          iiifUrl +
-          '" title="' +
-          this.content.iiif +
-          '" target="_blank"></a>'
-      );
-      this.$footer.append(this.$iiifButton);
-    }
-
-    this.$termsOfUseButton = $(
-      '<a href="#">' + this.extension.data.config!.content.termsOfUse + "</a>"
-    );
-    this.$footer.append(this.$termsOfUseButton);
-
-    this.$widthInput.on("keydown", (e) => {
-      return Numbers.numericalInput(e);
-    });
-
     this.$heightInput.on("keydown", (e) => {
       return Numbers.numericalInput(e);
     });
-
-    this.$shareInput.focus(function () {
-      $(this).select();
+    this.$heightInput.change(() => {
+      this.updateWidthRatio();
```

Comment:
I think something is broken with regard to this custom height/width embed code. When I switch the size to custom and try to change the dimensions, it seems to always get forced to a 1:1 ratio (i.e. setting one box sets both boxes). At universalviewer.dev, a different aspect ratio is getting enforced. I'm not sure why we can't accept arbitrary inputs, but in any case, I think something is wrong!